Raiblocks/Extensions/String+Extensions.swift

```diff
@@ -34,4 +34,7 @@ extension String {
         }
     }
 
+    func localized(comment: String = "") -> String {
+        return NSLocalizedString(self, comment: comment)
```

Comment:
Is this what's looking at the system language setting? How is Localizable.strings being used?

Reply:
Exactly. You can read this article, which explains better how it works: https://medium.com/lean-localization/ios-localization-tutorial-938231f9f881